app/app.go

```diff
@@ -647,10 +647,21 @@ func NewStrideApp(
 
 	// Add wasm keeper and wasm client keeper (must be after IBCKeeper and TransferKeeper)
 	wasmContractMemoryLimit := uint32(32)
-	wasmCapabilities := "iterator,staking,stargate,cosmwasm_1_1,cosmwasm_1_2,cosmwasm_1_3,cosmwasm_1_4,cosmwasm_2_0"
+	wasmCapabilities := []string{
+		"iterator",
+		"staking",
+		"stargate",
+		"cosmwasm_1_1",
+		"cosmwasm_1_2",
+		"cosmwasm_1_3",
+		"cosmwasm_1_4",
+		"cosmwasm_2_0",
+		"cosmwasm_2_1",
+		"cosmwasm_2_2",
```

Comment:
do you understand what these are for? I've never known lol

Reply:
those are features that contracts can use. e.g. if a contract would try to iterate over its state without the "iterator" feature being enabled, cw will fail the tx.

Reply:
what are the different cosmwasm_X_Y features?

Reply:
https://github.com/CosmWasm/cosmwasm/blob/release/2.2/packages/std/Cargo.toml#L20-L55